README.md

```diff
@@ -358,12 +357,13 @@ NamedColor   : Translucent
 The generated code will look like this:
 
 ```swift
-extension UIColor {
+// The Color type below is typealias'ed to UIColor on iOS and NSColor on OSX
+extension Color {
   /* Private Implementation details */
   ...
 }
 
-enum ColorName : UInt32 {
+enum ColorName {
```

Comment:
😆 In fact I had updated the body, bu when I was review my changes, I though their was voluntary more colors. On my way to fix this.